File: csharp/dsf/dsf/src/core/TaskCallException.cs
using System;

namespace com.baidu.dsf.core
{

    /// 
    /// <summary>
    /// @author arksea
    /// </summary>
    public class TaskCallException : Exception
    {

        public TaskCallException(string name, Exception ex) : base(name, ex)
        {
        }

        public TaskCallException(string name) : base(name)
        {
        }
    }

}

File: java/dsframework/src/main/java/dsf/core/TaskCallException.java
package dsf.core;

/**
 *
 * @author arksea
 */
public class TaskCallException extends RuntimeException {

    public TaskCallException(String name, Throwable ex) {
        super(name, ex);
    }

    public TaskCallException(String name) {
        super(name);
    }
}
